WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -212,6 +204,26 @@ private void unpackMediaIntent(@NonNull Intent intent) {
         }
     }
 
+    private void registerPostModelsForMedia(List<MediaModel> mediaList, boolean isRetry) {
+        if (mediaList != null && !mediaList.isEmpty()) {
+            Set<PostModel> postsToRefresh = PostUtils.getPostsThatIncludeThisMedia(mPostStore, mediaList);
```

Comment:
`PostUtils.getPostsThatIncludeThisMedia(mPostStore, mediaList)` - Should we rename this function?
From the name of the function it seems to me that it gets a single media in input, instead it's a list of media.

Checking the code of it, it adds posts that contain at least one media available in the list.

Reply:
Good call, renamed method to `getPostsThatIncludeAnyOfTheseMedia` in 40566eb